src/ir/context.rs

```diff
@@ -799,6 +799,23 @@ impl BindgenContext {
         }
     }
 
+    /// Assign a new generated name for each anonymous field.
+    fn deanonymize_fields(&mut self) {
+        for item in self.items.values_mut() {
```

Comment:
We should add `let _t = self.timer("deanonymize_fields");` at the top of this function.

Reply:
yeah sure!